app/lookups/land/event_type.rb

```diff
@@ -0,0 +1,9 @@
+module Land
+  class EventType < ApplicationRecord
+    include TableName
+
+    lookup_by :event_type, cache: 100
```

Comment:
Only this one isn't `find_or_create`.  I'm assuming it's intentional, so maybe we need to add how to add event types to the README or somewhere.